### PR TITLE
zk cache tree 's pathnode add path attribute and unit test

### PR DESCRIPTION
**Pull request by saaavsaaa**
Fixes #717 .

Changes proposed in this pull request:
-zk cache tree 's pathnode add path attribute
-cache unit test
-

### CODE REVIEW

**Comment by terrymanu** [comment]
please use assertThat replace of assert

**Comment by saaavsaaa** [comment]
The problem is not recurred. I'll check it first.

File: sharding-jdbc-orchestration/src/main/java/io/shardingsphere/jdbc/orchestration/reg/newzk/client/action/IClient.java [hunk 1]
```diff
@@ -50,6 +50,16 @@
...
     
+    /**
+     * block until connected.
+     *
+     * @param wait wait
+     * @param units units
+     * @return connected
+     * @throws InterruptedException InterruptedException
+     */
+    boolean blockUntilConnected(int wait, TimeUnit units) throws InterruptedException;
```
[the remark this answers is not in the document]
Comment (saaavsaaa) [reply sharding-jdbc-orchestration/src/main/java/io/shardingsphere/jdbc/orchestration/reg/newzk/client/action/IClient.java]: The "blockUntilConnected" is a public method. And when I solve the todo which on "boolean start(int wait, TimeUnit units)"，I will expire it.

File: sharding-jdbc-orchestration/src/main/java/io/shardingsphere/jdbc/orchestration/reg/newzk/client/retry/Callable.java [hunk 2]
```diff
@@ -30,6 +30,7 @@
  * @author lidongbo
  */
 public abstract class Callable<T> {
+    
     private static final Logger LOGGER = LoggerFactory.getLogger(Callable.class);
```
[the remark this answers is not in the document]
Comment (saaavsaaa) [reply sharding-jdbc-orchestration/src/main/java/io/shardingsphere/jdbc/orchestration/reg/newzk/client/retry/Callable.java]: rename to RetryCallable. Already use lombok , the result property use the @Setter but getResult have other logic.

**Comment by terrymanu** [comment]
just make sure all test case can be pass fisrt

**Comment by saaavsaaa** [comment]
I'll check it carefully.It passed in my fork branch.

**Pull request merged**